orchestrator/crates/prover-clients/atlantic-service/src/error.rs

```diff
@@ -34,7 +34,7 @@ pub enum AtlanticError {
     TaskIdSplit,
 
     #[error("Failed to encode PIE")]
-    PieEncode(#[source] starknet_os::error::SnOsError),
+    PieEncode(String),
```

Comment:
nitpick, Why are we replacing the error message instead of appending the actual error context?

```suggestion
#[error("Failed to encode PIE - {}")]
    PieEncode(#[source] starknet_os::error::SnOsError),
```

Reply:
The old error isn't available anymore. I have appended the string error for now